frontend/src/components/pod/List.tsx

```diff
@@ -233,8 +235,9 @@ export function PodListRenderer(props: PodListProps) {
         {
           id: 'status',
           gridTemplate: 'min-content',
+          filterVariant: 'multi-select',
           label: t('translation|Status'),
-          getValue: pod => getPodStatus(pod) + '' + pod.getDetailedStatus().reason,
+          getValue: pod => getPodStatus(pod) + '/' + pod.getDetailedStatus().reason,
```

Comment:
Maybe exporting this logic to another function so you can assign the pod status value and check it before the concatenation is a good idea. To avoid the Pending getting a prefix slash:
<img width="335" height="280" alt="image" src="https://github.com/user-attachments/assets/6caabb3d-38b8-4870-b42b-3fae4f9977b1" />